auth/src/main/java/com/firebase/ui/auth/ui/idp/WelcomeBackIdpPrompt.java

```diff
@@ -128,7 +128,16 @@ protected void onCreate(Bundle savedInstanceState) {
         mProvider.getOperation().observe(this, new Observer<Resource<IdpResponse>>() {
             @Override
             public void onChanged(Resource<IdpResponse> resource) {
```

Comment:
Fine for now, but we should probably invest in our own `Observer` class that abstracts some of this boilerplate.  We almost always do the same things.

Reply:
Yeah, even IntelliJ is getting fussy:
![image](https://user-images.githubusercontent.com/9490724/37852555-4e9c94b8-2ea0-11e8-9290-36ce55783130.png)

Reply:
Damn you have a fancier IntelliJ than I have!  I guess you don't use Android Studio?

Reply:
Haha, yeah, my daily driver is IntelliJ EAP because of stuff like that. (Of course, I do have Android Studio Canary so I can hop over there if I need to. 😁)